dev-libs/zug/metadata.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
+		<email>kde@gentoo.org</email>
+		<name>Gentoo KDE Project</name>
```

Comment:
KDE proj can't pick up any new packages with current team size.

Are you already proxy-maintaining packages or would you like to volunteer for this?

Reply:
I am already proxy maintainer for a few packages. As you though, I'm also a bit pressed on resources.
But if you are OK with that I can add myself here instead